src/components/Header/Header.module.css

```diff
@@ -0,0 +1,17 @@
+.loadFactoryHeader {
+  padding-bottom: 0;
+  user-select: none;
```

Comment:
What is the motivation to prevent user to select something in component's header?

Reply:
Ok. Just added the same what we have for the workspace details header. I will remove it in both places.

Reply:
I have fixed it.